nibabel/cifti2/__init__.py

```diff
@@ -0,0 +1,26 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the NiBabel package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""CIfTI format IO
+
+.. currentmodule:: nibabel.cifti2
```

Comment:
@matthew-brett - these were copied over from the prior `__init__` files that things were tracked from: `gifti, nicom`. if there is no need for this perhaps that can be handled in a separate PR going across nibabel.

is there an automated documentation builder somewhere for pull-requests and the main repo?